styles/library.scss

```diff
@@ -11,7 +11,11 @@
     }
 
     :where(ul):not(:where([class~='unstyled-text'] *)) {
-      @apply list-disc list-outside pl-4 space-y-2;
+      @apply list-disc list-outside ml-8 space-y-2;
+    }
+
+    :where(ol):not(:where([class~='unstyled-text'] *)) {
```

Comment:
Is `:where` being used here for its specificity behavior?

Reply:
Yep.